### DISCUSSION TITLE
Request: long lasting URLs for Algerian channels

### DISCUSSION THREAD

**Discussion by ARAB-IPTV** [iptv-org/community#166] [Q&A]
hello .
if you can make a long time url for algerians channels from the site htatv.com 
like this :
[[http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.htatv.com/channel/6

**Comment by ARAB-IPTV** [comment]
is that impossible ! answer me ...

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/master/streams/dz.m3u
